src/RSocketClient.cpp

```diff
@@ -41,11 +44,11 @@ folly::Future<std::unique_ptr<RSocketRequester>> RSocketClient::connect(
     keepaliveTimer = std::move(keepaliveTimer),
     stats = std::move(stats),
     networkStats = std::move(networkStats),
-    promise = std::move(promise)](
-      std::unique_ptr<DuplexConnection> connection,
-      folly::EventBase& eventBase) mutable {
+    promise = std::move(promise)
+  ](std::unique_ptr<DuplexConnection> connection,
+    folly::EventBase & eventBase) mutable {
     VLOG(3) << "onConnect received DuplexConnection";
-
+    evb_ = &eventBase;
```

Comment:
Threading is supposed to all be handled at only a single layer, by the "Scheduled" wrappers here: https://github.com/rsocket/rsocket-cpp/tree/master/src/internal

So why is this needed?